### DISCUSSION TITLE
[Closed] Possible new converter to check for null

### DISCUSSION THREAD

**Discussion by IeuanWalker** [CommunityToolkit/Maui#305] [New Feature Discussions]
Before this change #289 I was using the converter to check if `double?` was null, but due to that change, the converter now throws an `InvalidCastException` if it's not a `string`.

So I propose we create some new converters to cover the old behaviour `IsNullConverter` & `IsNotNullConverter`

**Comment by TheCodeTraveler** [comment]
Thanks @ieuanwalker!

This was one of the scenarios that we were concerned about breaking when re-implementing `IsStringNullOrEmptyConverter`. 

I think `IsNullConverter` and `IsNotNullConverter` are good names for the new converters 👍 

If you're able to open a New Feature Proposal for this I'd be happy to Champion it and bring it up for a vote/approval in our next standup 💯: 

https://github.com/CommunityToolkit/Maui/issues/new?assignees=&labels=new%2C+proposal&template=open-a-new-feature-proposal.md&title=%5BProposal%5D+

**Reply by IeuanWalker** [reply]
Thanks @brminnick, done #306

**Comment by CliffAgius** [comment]
Closed as completed #306
